src/stratum_server.cpp

```diff
@@ -525,6 +526,8 @@ void StratumServer::on_blobs_ready()
 		}
 	}
 
+	api_update_local_stats();
```

Comment:
This should be moved to `StratumServer::on_share_found`, right after `server->update_hashrate_data` call. But it should also be rate-limited: remember the timestamp when it was last called and call it once a minute.